datasetconfig.json

```diff
@@ -492,14 +521,14 @@
         "cache": 6,
         "climatology": "http://navigator.oceansdata.ca:8080/thredds/dodsC/climatology/Levitus98_PHC21/aggregated.ncml",
         "attribution": "The Canadian Centre for Meteorological and Environmental Prediction",
-        "lat_var_key": "y",
-        "lon_var_key": "x",
+        "lat_var_key": "nav_lat",
+        "lon_var_key": "nav_lon",
         "help": " ",
 
         "variables": {
             "somecrty": { "name": "ocean current along j-axis", "envtype": "ocean", "unit": "m/s", "scale": [-1, 1], "zero_centered": "true" },
             "sozocrtx": { "name": "ocean current along i-axis", "envtype": "ocean", "unit": "m/s", "scale": [-1, 1], "zero_centered": "true" },
-            "magwatervel": { "name": "ocean current magnitude", "envtype": "ocean", "unit": "m/s", "scale": [0, 1], "equation": "magnitude(sozocrtx, somecrty)",  "dims": ["time_counter", "y", "x"] },
+            "magwatervel": { "name": "ocean current magnitude", "envtype": "ocean", "unit": "m/s", "scale": [0, 1], "equation": "magnitude(sozocrtx, somecrty)",  "dims": ["time_counter", "y", "x"], "east_vector_component": "vozocrtx", "north_vector_component": "vozocrty" },
```

Comment:
For GSL (2D and 3D), vozocrtx/vomecrty and somecrty/sozocrtx are not eastward/northward. You need to calculate the rotated components (vozocrte/vomecrtn). See the other datasets for examples. These new variables should be used as east_vector_component/north_vector_component. There is a grid_angle.nc file for gsl available in the /data/grids directory.